common/anomalies/hfe_anomaly.txt

```diff
@@ -228,18 +232,11 @@ hfe_anomaly_blight = {
 		modifier = {
 			factor = 0.1
 			exists = from.owner
+			NOT = { hfe_trigger_hardcore_enabled = yes } 
```

Comment:
Just use
hfe_trigger_hardcore_enabled = no

Reply:
_Custom_ Scripted Triggers are different from Vanilla rendition Scripted Triggers. Not all of them accept both Boolean/binary commands. NOT/YES is important for _custom_ Scripted Triggers.

Reply:
No, it always work fine

Reply:
I've tried it before. It doesn't always work.

Reply:
It always work in stellaris

Reply:
It's fine as it is.

Reply:
No

Reply:
You completely misunderstood why I did this.

I initially removed this entirely, but you said that you wanted the player and the AI to have an equal chance to get Blighted.

When Hardcore Mode is disabled, the player _cannot be Blighted_. Therefore, the AI is the only eligible candidate. That is why the AI only gets this in Hardcore Mode, when it competes with the player to "win" the Blight.

Reply:
Equal chance will be without this line.
If hadrcore disabled, ai will get 0.1 multiplicator.
What this mean? Less chance to get anomaly for ai. Still its is intended as something rare.
If hardcore enabled, ai will not get multiplicator.
What this mean? Player never will be blight.
Because if you have 15 ais in game, your chance 1 to 15.

Reply:
THE PLAYER CANNOT GET BLIGHTED IF HARDCORE MODE IS DISABLED. ONLY IN HARDCORE MODE CAN THE PLAYER GET BLIGHTED.

Reply:
Okay, done.